### DISCUSSION TITLE
Where is the demo of explorer?

### DISCUSSION THREAD

**Discussion by supemeko** [emilk/egui#3209] [Q&A]
![image](https://github.com/emilk/egui/assets/38853195/0e5b1e3b-d275-4596-8670-14f0e1b7efb1)
like this

**Comment by supemeko** [comment]
Is there such a widget in egui, if so, please tell me where to find it?

**Comment by stfnp** [comment]
Hi @supemeko, where did you find that screenshot? I would be interested too.

As far as I know there is no dedicated tree widget in egui (yet?), but something similar can be achieved by nesting [CollapsingState](https://docs.rs/egui/latest/egui/containers/collapsing_header/struct.CollapsingState.html) and putting a [SelectableLabel](https://docs.rs/egui/latest/egui/struct.Ui.html#method.selectable_label) or [SelectableValue](https://docs.rs/egui/latest/egui/struct.Ui.html#method.selectable_value) into the headers. The screenshot doesnt look like that though.

**Reply by Mingun** [reply]
This is VS Code

**Reply by stfnp** [reply]
Oh, you're right. I thought it was some egui demo.
